tests/pipeline-all-amps/repo/Dockerfile

```diff
@@ -24,9 +24,6 @@ USER root
 ARG TOMCAT_DIR=/usr/local/tomcat
 ARG GROUPANME=Alfresco
 
-# Copy the idp.jks keystore file used to enable AOS with SAML
-COPY idp.jks /usr/src/alfresco/
```

Comment:
Can we remove `idp.jks` file from the repo directory if it is not needed anyway ?